### PR TITLE
feat: support pg type alias

### PR DESCRIPTION
**Pull request by Kikkon**
fix: https://github.com/sqlparser-rs/sqlparser-rs/issues/931

### CODE REVIEW

**Comment by Kikkon** [comment]
I'd appreciate it if you could give me some feedback on this PR whenever you have time. @alamb

**Comment by alamb** [comment]
Thanks -- I will try and get to it later this week @Kikkon

File: src/ast/data_type.rs [hunk 1]
```diff
@@ -109,27 +116,50 @@ pub enum DataType {
...
+    /// Unsigned int4 with optional display width e.g. INT4 UNSIGNED or INT4(11) UNSIGNED
+    UnsignedInt4(Option<u64>),
     /// Unsigned integer with optional display width e.g. INTGER UNSIGNED or INTEGER(11) UNSIGNED
     UnsignedInteger(Option<u64>),
     /// Big integer with optional display width e.g. BIGINT or BIGINT(20)
     BigInt(Option<u64>),
     /// Unsigned big integer with optional display width e.g. BIGINT UNSIGNED or BIGINT(20) UNSIGNED
     UnsignedBigInt(Option<u64>),
+    /// Int8 as alias for Bigint in [postgresql]
+    /// Note: Int8 mean 8 bytes in postgres (not 8 bits)
```
Comment (alamb) [inline src/ast/data_type.rs:139]: thank you for the nice comments

**Review by alamb — COMMENTED** [review]
Hi @Kikkon  -- thank you for the contribution. I think this looks like a good approach and I especially like that it is fairly backwards compatible (doesn't change existing variants, just adds new ones)

It looks like there are some CI tests to fix but then this one should be good to go!

**Comment by Kikkon** [comment]
> Hi @Kikkon -- thank you for the contribution. I think this looks like a good approach and I especially like that it is fairly backwards compatible (doesn't change existing variants, just adds new ones)
> 
> It looks like there are some CI tests to fix but then this one should be good to go!

Already fixed the CI tests.

**Review by alamb — APPROVED** [review]
Thanks @Kikkon

**Pull request merged**